### ISSUE TITLE
[loss function] beta loss function

### ISSUE THREAD

**Issue by deebuls** [deebuls/devblog#18]
Inspired by the dirichlet loss function .

We can have the beta loss function . 

* for each encoded output 2 values \alpha and \beta will be predicted
* For expected 0 higher value of \alpha is expected and for expected higher value of \beta.
* Probable advantage over the dirichlet representation is that the zero value can also contribute by going as big number.
* Combining this with ECOC hadmard encoding will lead to a more robust representation.
* Calculating uncertainty will be averaging the entropy of all the individual digits in the hadmard code.

**Comment by deebuls** [comment]
Preliminary  and PyTorch code

https://colab.research.google.com/drive/1RYkiFjNPyLcUpRn4amSBYBSToLB2P2OW

Biggest issue is with multiple classes the number of zeros is very high with respect to ones . so need to check. 

Best suited with ECOC encoding .

What about the activation function ?  Algebriac sigmoid , sparsemax
